README.md

```diff
@@ -56,16 +59,101 @@ and migrate:
 rake db:migrate
 ```
 
-Also include `BookingSync::Engine::Account` in your `Account` model:
+Also include `BookingSync::Engine::Models::Account` in your `Account` model:
+
+```ruby
+class Account < ActiveRecord::Base
+  include BookingSync::Engine::Models::Account
+end
+```
+
+When saving new token, this gem uses a separate thread with new db connection to ensure token save (in case of a rollback in the main transaction).
```

Comment:
Is it still the case though?  I think we might have removed it some time ago

Reply:
I don't think it's the case.
I don't see any threading in the code.